cumulusci/core/utils.py

```diff
@@ -1,6 +1,45 @@
+""" Utilities for CumulusCI Core
+
+import_class: Task class defn import helper
+MockLoggingHandler: a handler for testing logging """
+
+import logging
+
+
 def import_class(path):
+    """ Import a class from a string module class path """
     components = path.split('.')
     module = components[:-1]
     module = '.'.join(module)
     mod = __import__(module, fromlist=[components[-1]])
     return getattr(mod, components[-1])
+
+
+class MockLoggingHandler(logging.Handler):
```

Comment:
moved it into cumulusci.core.tests.utils, since it doesn't actually contain tests to run.